src/com/goide/psi/impl/GoTypeUtil.java

```diff
@@ -41,6 +41,13 @@ private static boolean isArray(@Nullable GoType type) {
 
   public static boolean isString(@Nullable GoType type) {
     GoType underlyingType = type != null ? type.getUnderlyingType() : null;
-    return underlyingType != null && underlyingType.textMatches("string") && GoPsiImplUtil.builtin(underlyingType);
+    return underlyingType != null &&
+           underlyingType.getText() != null &&
```

Comment:
maybe it makes sense to implement `textMatches` or `getText` methods for CTypes here. Or maybe check GoCType explicitly like `underlyingType instanceof GoCType`. Anyway test is required.